mailbox/jpa/pom.xml

```diff
@@ -89,6 +101,16 @@
             <artifactId>james-server-data-jpa</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-jpa-common-guice</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-jpa-common-guice</artifactId>
+        </dependency>
```

Comment:
Why is this dependency needed?

Reply:
This dependency was added in order to enable the usage of JPAConfiguration. Do you suggest an alternative approach?

Reply:
Move JPAConfiguration to backends/jpa